BookingsApi/BookingsApi.DAL/Mappings/JudiciaryPersonMap.cs

```diff
@@ -0,0 +1,28 @@
+using BookingsApi.Domain;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace BookingsApi.DAL.Mappings
+{
+    public class JudiciaryPersonMap : IEntityTypeConfiguration<JudiciaryPerson>
+    {
+        public void Configure(EntityTypeBuilder<JudiciaryPerson> builder)
+        {
+            builder.ToTable(nameof(JudiciaryPerson));
+
+            builder.HasKey(x => x.Id);
+            builder.Property(x => x.Id).ValueGeneratedNever();
+            builder.Property(x => x.ExternalRefId);
```

Comment:
Since the query uses the `ExternalRefId` to find a judiciary person I think we should index this property

Reply:
Done